app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -1,6 +1,22 @@
 <ng-form name="persistentVolumeClaimForm">
   <fieldset ng-disabled="claimDisabled">
-
+    <!--storage class-->
+    <label class="required">Storage Classes</label>
+    <div>
+      <span id="claim-storage-class-help" class="help-block">Storage classes are set by the administrator to define types of storage the users can select.
+       <div class="learn-more-block" ng-class="{ 'gutter-bottom': showPodWarning }">
+        <a href="{{'storage_classes' | helpLink}}" target="_blank">Learn more <i class="fa fa-external-link" aria-hidden> </i></a>
+       </div>
+      </span>
+    </div>
+    <div class="form-group" ng-repeat="sclass in storageClasses" id="storageclass-{{sclass.label}}">
+      <input  type="radio"
+              name="storageClass"
+              value="{{sclass.label}}"
+              ng-model="claim.storageClass"
+              ng-change="handleStorageClassClick(sclass)">
+      {{sclass.label}}<span class="help-block">{{sclass.description != "" && sclass.description != null ? sclass.description : ""}}</span>
```

Comment:
Probably want this for the radios, although I'd like to see how the help blocks look.

```html
<div class="radio">
  <label>
    <input type="radio"
      name="storageClass"
      value="{{sclass.label}}"
      ng-model="claim.storageClass"
      ng-change="handleStorageClassClick(sclass)">
    {{sclass.label}}
  </label>
  <div ng-if="scalss.description" class="help-block">{{sclass.description}}</div>
</div>
```